link/uprobe_multi_test.go

```diff
@@ -0,0 +1,133 @@
+package link
+
+import (
+	"errors"
+	"os/exec"
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal/testutils"
+	"github.com/cilium/ebpf/internal/unix"
+)
+
+func TestUprobeMulti(t *testing.T) {
+	testutils.SkipIfNotSupported(t, haveBPFLinkUprobeMulti())
```

Comment:
Nit: it's cleaner to do this on the `err` returned from UprobeMulti.

Reply:
ok, will change

Reply:
looks like this needs to be the first line of the test after all.. I'm getting error on 4.9 kernel because of the unknown attach type.. so the test will fail on loading the test program and fail